### PR TITLE
moved nightly_boards folder to deskdaily and edited keyboard aliases

### PR DESCRIPTION
**Pull request by DeskDaily**
I renamed my vendor folder from "nightly_boards" to "deskdaily" I also update `data/mappings/keyboard_aliases.hjson`  by adding my boards at the end. Let me know if this is not the proper way.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: data/mappings/keyboard_aliases.hjson [hunk 1]
```diff
@@ -1312,5 +1312,51 @@
     },
     "zinc/reva": {
         "target": "25keys/zinc/reva"
+    },
+    // Moved nightly_boards to deskdaily
```
Comment (dunk2k) [inline data/mappings/keyboard_aliases.hjson:1316]:
```suggestion
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: data/mappings/keyboard_aliases.hjson
Comment (dunk2k) [inline data/mappings/keyboard_aliases.hjson:1]: entries here to be added in alphabetical order

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: data/mappings/keyboard_aliases.hjson
Comment (DeskDaily) [reply data/mappings/keyboard_aliases.hjson:1]: Thanks. But do I need to add it towards the end?

**Comment by DeskDaily** [comment]
> opportunity to move `manufacturer`, `maintainer`, and `usb.vid` from each board's `info.json` into (a vendor master) `keyboards/deskdaily/info.json` 😉

do you have an example of how that will look like?

File: data/mappings/keyboard_aliases.hjson
Comment (dunk2k) [reply data/mappings/keyboard_aliases.hjson:1]: > Thanks. But do I need to add it towards the end?

not towards the end, alphabetically with(in) the rest of this file's contents/entries.

**Comment by dunk2k** [comment]
> > opportunity to move `manufacturer`, `maintainer`, and `usb.vid` from each board's `info.json` into (a vendor master) `keyboards/deskdaily/info.json` 😉
> 
> do you have an example of how that will look like?

```json
{
    "manufacturer": "",
    "maintainer": "",
    "usb" : {
        "vid": "",
    }
}
these fields would be removed from every member/child keyboard's `info.json`
```

**Pull request closed without merging**